src/Raven.Client/ServerWide/DatabaseTopology.cs

```diff
@@ -140,6 +140,17 @@ public override int GetHashCode()
         }
     }
 
+    public static class ThreadSafeRandom
```

Comment:
Can we change it to `private` or `internal`? I don't want it to be considered public API

Reply:
done